CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
-# Changelog
-All notable changes to this project will be documented in this file.
+# Change Log
+
+All notable changes to the Zowe CLI package will be documented in this file.
+
+## `6.9.2`
+
+- Return non-zero exit code when upload command fails. Thanks @tjohnsonBCM
+
+## `6.9.1`
+
+- Support `#` in account name supplied to TSO commands. Thanks @awharn
+
+## `6.9.0`
+
+- Add API to recall migrated datasets. Thanks @Pranay154
+
+## `6.8.2`
+
+- Update the Zowe logo to the new logo. Thanks @awharn
+
+## `6.8.1`
+
+- Add utility function to access ImperativeConfig. Thanks @tjohnsonBCM
+
+## `6.8.0`
+
+- Add possibility to use Etags with download and upload APIs. Thanks @Alexandru-Dimitru
+- Add option to return Etag on upload. Thanks @Alexandru-Dimitru
+
+## `6.0.0`
+
+- Rename `files list zfs` command to `files list fs` since it is not specific to ZFS filesystems.
+
+## `5.0.0`
+
+- Use new streaming RestClient APIs to reduce memory usage when downloading and uploading files.
+
+## `4.0.0`
+
+- Remove the method `Get.dataSetStreamed`. Use `ZosmfRestClient.getStreamed` instead.
+
+## `3.0.0`
+
+- Rename package from "@brightside/core" to "@zowe/cli".
+- Change behavior of the method `Shell.executeSsh` to use `stdoutHandler` instead of `streamCallBack`. This eliminates dependency on the `ClientChannel` type of the ssh2 package.
 
 ## [2.0.0] prerelease -2019-1-11
```

Comment:
Should I remove the pre-zowe-v1-lts changelog from this line and below in the file, or leave it?